frontend/src/common/index.ts

```diff
@@ -9,4 +9,4 @@ import * as FeedMessage from './feed';
 export { Types, FeedMessage };
 
 // Increment this if breaking changes were made to types in `feed.ts`
-export const VERSION: Types.FeedVersion = 31 as Types.FeedVersion;
+export const VERSION: Types.FeedVersion = 32 as Types.FeedVersion;
```

Comment:
What happens to a v31 browser connecting to a v32 feed? Will it know to reload itself? (Would be good to add to the comment what the behaviour is here)

Reply:
@dvdplm will include a comment in the backport (frontport?) to master (and yes, it will reload itself on mismatch)